Abubakarharuna10.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "@Abubakarharuna10"
+project: "Artificial Rescuer"
+---
+
+# Artificial Rescue
+
+## Summary
+
+With the increasing threat of climate change, greater populations and consequently, more extreme environmental events
+the need for better emergency rescue services is paramount
+As time moves forward, natural disasters will continue to wreak havoc on cities and counties around the world, 
+causing many people to die. Bolstering the power of search and rescue teams with autonomous UAVs that can identify people
+in disaster relief zones could significantly reduce the impact tornadoes, earthquakes, tsunamis, floods, etc...
+My solution to strengthening search and rescue operations is to outfit an autonomous unmanned aerial vehicle (UAV) 
+with a computer vision system that detects the location of people as the vehicle flies over ground. The powerful neural-network capabilities
+of the Jetson Nano Dev Kit will enable fast computer vision algorithms to achieve this task. Joseph Redmon's YOLOv3 algorithm will be used 
+to do the actual object-detection (people) in the camera's view While the UAV is flying a waypoint mission using ArduPilot, PX4, or any other autonomous flight control stack, the absolute location of people in the camera view can be calculated based on the altitude, orientation, and GPS location of the UAV.
+
+## Plan
+
+
+For the first day I will make all the neccessary equipment I would need to use. and the next 2 or 3 days I will write the programming code and build the AI model .
+The rest of the days will be spent constructing and testing the project.
+
+I will use  NVIDIA Jetson Nano Developer Kit to run computer vision algorithms to detect people as the vehicle flies overhead. I will use an Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter that will allow me to connect to the Jetson Nano and monitor processes while the vehicle is in a flight setting. The CanaKit 5V 2.5A Power Supply will provide a stable power source while setting up the Jetson Nane. The Raspberry Pi Camera Module V2 will be mounted to the underside of the vehicle to capture video of the ground below. As for the Anker PowerCore 5000, this power pack was selected for its 2A output and compact size. Then, the PX4 Pixhawk on an autonomous drone or airplane will communicate with the Jetson Nano over a wired MAVLink connection  but you can use any autonomous drone or airplane in order to Mount your work on it.
```

Comment:
Do you have an autonomous drone or airplane to mount your project on? No worries if you don't. I am curious as to what your final project might look like. Will you be mounting your system onto a drone, or just building the detection system?

Reply:
yes I will try to own it before ending of the  project else I will collect my friend's own to use for demonstration

Thank you!